src/couch/src/couch_stream.erl

```diff
@@ -259,7 +259,7 @@ handle_call({write, Bin}, _From, Stream) ->
                         buffer_len=0,
                         md5=Md5_2,
                         identity_md5=IdenMd5_2,
-                        identity_len=IdenLen + BinSize}};
+                        identity_len=IdenLen + BinSize}, hibernate};
```

Comment:
@savanmorya There's a known issue with some uses of large (> 64bits) binaries when the process using them doesn't do much work. Here are two good writeups:

http://blog.bugsense.com/post/74179424069/erlang-binary-garbage-collection-a-lovehate
https://blog.heroku.com/logplex-down-the-rabbit-hole

The reason for hibernate is that it forces a full garbage collection where as `erlang:garbage_collect(self())` won't always clean out all binary usage. As Fred says its the least ugly solution when its available and the processes are known.